cms/envs/common.py

```diff
@@ -751,7 +751,7 @@
 
     # Tracking
     'track',
-    'eventtracking.django',
+    'eventtracking.django.apps.EventTrackingConfig',
```

Comment:
Oh roger, I misunderstood. This is new in Django -- the other way to do this would have been to specify a default in the eventtracking package.